app/js/services/tableau.js

```diff
@@ -91,11 +91,21 @@ const getParameters = (workbook) => getWorkbookParameters(workbook).then((tablea
 
 /**
  * Returns formatted tableau filters as application filters
+ *
+ * This currently just fetches the filters from Tableau and trkrefs from current_user's profile
+ * It should also fetch the filters we need from the client_portal-analytics_backend in order to
+ * correctly select product names/etc.
```

Comment:
Is this comment the change we made or a further change that needs to be made? If it's what it does with this change the wording needs to be different.